### ISSUE TITLE
qt/6.0.0-beta2: Error finding dependancies during package build.

### ISSUE THREAD

**Issue by Dark-Passenger** [bincrafters/community#1293]
### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **qt/6.0.0-beta2**
  * Operating System+version: **Windows 10**
  * Compiler+version: **Visual studio 2019**
  * Docker image: **conanio/gcc8**
  * Conan version: **conan 1.18.0**
  * Python version: **Python 3.7.4**

```bash
qt/6.0.0-beta2@bincrafters/testing: Calling build()
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.7.7
** Copyright (c) 2020 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
Conan:vcvars already set
-- The CXX compiler identification is MSVC 19.27.29112.0
-- The C compiler identification is MSVC 19.27.29112.0
-- The ASM compiler identification is MSVC
-- Found assembler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Extracting architecture info from C:/.conan/9cd064/1/qtbase/config.tests/arch/architecture_test.exe.
-- Performing Test HAVE_LD_VERSION_SCRIPT
-- Performing Test HAVE_LD_VERSION_SCRIPT - Failed
-- CMAKE_HOST_SYSTEM: "Windows-10.0.19041"
-- CMAKE_HOST_SYSTEM_NAME: "Windows"
-- CMAKE_HOST_SYSTEM_VERSION: "10.0.19041"
-- CMAKE_HOST_SYSTEM_PROCESSOR: "AMD64"
-- CMAKE_SYSTEM: "Windows"
-- CMAKE_SYSTEM_NAME: "Windows"
-- CMAKE_SYSTEM_VERSION: "10.0.19041"
-- CMAKE_SYSTEM_PROCESSOR: "AMD64"
-- CMAKE_CROSSCOMPILING: "FALSE"
-- CMAKE_C_COMPILER: "C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe" (19.27.29112.0)
-- CMAKE_CXX_COMPILER: "C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe" (19.27.29112.0)
-- MSVC_VERSION: "1927"
-- MSVC_TOOLSET_VERSION: "142"
-- Conan: Using autogenerated FindZLIB.cmake
-- Found ZLIB: 1.2.11 (found suitable version "1.2.11", minimum required is "1.0.8")
-- Library zlib found C:/Users/Dhruv/.conan/data/zlib/1.2.11/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/zlib.lib
-- Found: C:/Users/Dhruv/.conan/data/zlib/1.2.11/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/zlib.lib
-- Could NOT find ZSTD (missing: ZSTD_DIR)
did not find zstd
-- Could NOT find ZSTD: Found unsuitable version "", but required is at least "1.3" (found ZSTD_LIBRARY-NOTFOUND)
-- Could NOT find WrapDBus1 (missing: DBus1_LIBRARY DBus1_INCLUDE_DIR WrapDBus1_FOUND) (Required is at least version "1.2")
-- Performing Test HAVE_cxx14
-- Performing Test HAVE_cxx14 - Success
-- Performing Test HAVE_cxx17
-- Performing Test HAVE_cxx17 - Success
-- Performing Test HAVE_cxx2a
-- Performing Test HAVE_cxx2a - Success
-- Performing Test HAVE_precompile_header
-- Performing Test HAVE_precompile_header - Failed
-- Performing Test TEST_use_bfd_linker
-- Performing Test TEST_use_bfd_linker - Failed
-- Performing Test TEST_use_gold_linker
-- Performing Test TEST_use_gold_linker - Failed
-- Performing Test TEST_use_lld_linker
-- Performing Test TEST_use_lld_linker - Failed
-- Performing Test TEST_optimize_debug
-- Performing Test TEST_optimize_debug - Failed
-- Performing Test TEST_enable_new_dtags
-- Performing Test TEST_enable_new_dtags - Success
-- Performing Test TEST_gdb_index
-- Performing Test TEST_gdb_index - Success
-- Performing Test HAVE_reduce_relocations
-- Performing Test HAVE_reduce_relocations - Failed
-- Performing Test separate debug information support
-- Performing Test separate debug information support - Failed
-- Performing Test HAVE_signaling_nan
-- Performing Test HAVE_signaling_nan - Success
-- Performing SIMD Test SSE2 instructions
-- Performing SIMD Test SSE2 instructions - Success
-- Performing SIMD Test SSE3 instructions
-- Performing SIMD Test SSE3 instructions - Success
-- Performing SIMD Test SSSE3 instructions
-- Performing SIMD Test SSSE3 instructions - Success
-- Performing SIMD Test SSE4.1 instructions
-- Performing SIMD Test SSE4.1 instructions - Success
-- Performing SIMD Test SSE4.2 instructions
-- Performing SIMD Test SSE4.2 instructions - Success
-- Performing SIMD Test AES new instructions
-- Performing SIMD Test AES new instructions - Success
-- Performing SIMD Test F16C instructions
-- Performing SIMD Test F16C instructions - Success
-- Performing SIMD Test RDRAND instruction
-- Performing SIMD Test RDRAND instruction - Success
-- Performing SIMD Test RDSEED instruction
-- Performing SIMD Test RDSEED instruction - Success
-- Performing SIMD Test SHA new instructions
-- Performing SIMD Test SHA new instructions - Success
-- Performing SIMD Test AVX instructions
-- Performing SIMD Test AVX instructions - Success
-- Performing SIMD Test AVX2 instructions
-- Performing SIMD Test AVX2 instructions - Success
-- Performing SIMD Test AVX512 F instructions
-- Performing SIMD Test AVX512 F instructions - Success
-- Performing SIMD Test AVX512 ER instructions
-- Performing SIMD Test AVX512 ER instructions - Success
-- Performing SIMD Test AVX512 CD instructions
-- Performing SIMD Test AVX512 CD instructions - Success
-- Performing SIMD Test AVX512 PF instructions
-- Performing SIMD Test AVX512 PF instructions - Success
-- Performing SIMD Test AVX512 DQ instructions
-- Performing SIMD Test AVX512 DQ instructions - Success
-- Performing SIMD Test AVX512 BW instructions
-- Performing SIMD Test AVX512 BW instructions - Success
-- Performing SIMD Test AVX512 VL instructions
-- Performing SIMD Test AVX512 VL instructions - Success
-- Performing SIMD Test AVX512 IFMA instructions
-- Performing SIMD Test AVX512 IFMA instructions - Success
-- Performing SIMD Test AVX512 VBMI instructions
-- Performing SIMD Test AVX512 VBMI instructions - Success
-- Performing Test HAVE_posix_fallocate
-- Performing Test HAVE_posix_fallocate - Failed
-- Performing Test HAVE_alloca_stdlib_h
-- Performing Test HAVE_alloca_stdlib_h - Failed
-- Performing Test HAVE_alloca_h
-- Performing Test HAVE_alloca_h - Failed
-- Performing Test HAVE_alloca_malloc_h
-- Performing Test HAVE_alloca_malloc_h - Success
-- Performing Test HAVE_stack_protector
-- Performing Test HAVE_stack_protector - Success
-- Performing Test HAVE_intelcet
-- Performing Test HAVE_intelcet - Failed
-- Could NOT find double-conversion (missing: double-conversion_DIR)
-- Could NOT find WrapDoubleConversion (missing: WrapDoubleConversion_FOUND)
-- Found GLIB2: C:/.conan/ba70c5/1/lib/glib-2.0.lib
-- Found the following ICU libraries:
--   i18n (required)
--   uc (required)
-- The following ICU libraries were not found:
--   data (required)
-- Failed to find all ICU components (missing: ICU_INCLUDE_DIR _ICU_REQUIRED_LIBS_FOUND)
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Performing Test HAVE_GETTIME
-- Performing Test HAVE_GETTIME - Failed
-- Could NOT find WrapRt (missing: WrapRt_FOUND)
-- Could NOT find LTTngUST (missing: LTTNGUST_LIBRARIES LTTNGUST_INCLUDE_DIRS)
-- Found WrapSystemPCRE2: C:/Users/Dhruv/.conan/data/pcre2/10.33/_/_/package/c3909bc59b2700f5c593b64eb3cdec0e85b05bc5/lib/pcre2-16.lib (Required is at least version "10.20")
-- Could NOT find Slog2 (missing: Slog2_INCLUDE_DIR Slog2_LIBRARY)
-- Performing Test HAVE_atomicfptr
-- Performing Test HAVE_atomicfptr - Success
-- Performing Test HAVE_cloexec
-- Performing Test HAVE_cloexec - Failed
-- Performing Test HAVE_cxx11_future
-- Performing Test HAVE_cxx11_future - Success
-- Performing Test HAVE_cxx11_random
-- Performing Test HAVE_cxx11_random - Success
-- Performing Test HAVE_cxx17_filesystem
-- Performing Test HAVE_cxx17_filesystem - Success
-- Performing Test HAVE_eventfd
-- Performing Test HAVE_eventfd - Failed
-- Performing Test HAVE_futimens
-- Performing Test HAVE_futimens - Failed
-- Performing Test HAVE_futimes
-- Performing Test HAVE_futimes - Failed
-- Performing Test HAVE_getauxval
-- Performing Test HAVE_getauxval - Failed
-- Performing Test HAVE_getentropy
-- Performing Test HAVE_getentropy - Failed
-- Performing Test HAVE_glibc
-- Performing Test HAVE_glibc - Failed
-- Performing Test HAVE_inotify
-- Performing Test HAVE_inotify - Failed
-- Performing Test HAVE_ipc_sysv
-- Performing Test HAVE_ipc_sysv - Failed
-- Performing Test HAVE_ipc_posix
-- Performing Test HAVE_ipc_posix - Failed
-- Performing Test HAVE_linkat
-- Performing Test HAVE_linkat - Failed
-- Performing Test HAVE_ppoll
-- Performing Test HAVE_ppoll - Failed
-- Performing Test HAVE_pollts
-- Performing Test HAVE_pollts - Failed
-- Performing Test HAVE_poll
-- Performing Test HAVE_poll - Failed
-- Performing Test HAVE_renameat2
-- Performing Test HAVE_renameat2 - Failed
-- Performing Test HAVE_statx
-- Performing Test HAVE_statx - Failed
-- Performing Test HAVE_syslog
-- Performing Test HAVE_syslog - Failed
-- Performing Test HAVE_xlocalescanprint
-- Performing Test HAVE_xlocalescanprint - Failed
-- Conan: Using autogenerated FindOpenSSL.cmake
-- Found OpenSSL: 1.1.1h (found version "1.1.1h")
-- Library libssl found C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libssl.lib
-- Found: C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libssl.lib
-- Library libcrypto found C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libcrypto.lib
-- Found: C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libcrypto.lib
-- Library libcrypto found C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libcrypto.lib
-- Found: C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libcrypto.lib
-- Library libssl found C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libssl.lib
-- Found: C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libssl.lib
-- Could NOT find WrapOpenSSLHeaders (missing: WrapOpenSSLHeaders_FOUND)
-- Conan: Using autogenerated FindOpenSSL.cmake
-- Library libssl found C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libssl.lib
-- Skipping already existing target: CONAN_LIB::OpenSSL_libssl
-- Found: C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libssl.lib
-- Library libcrypto found C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libcrypto.lib
-- Skipping already existing target: CONAN_LIB::OpenSSL_libcrypto
-- Found: C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libcrypto.lib
-- Library libcrypto found C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libcrypto.lib
-- Skipping already existing target: CONAN_LIB::OpenSSL_Crypto_libcrypto
-- Found: C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libcrypto.lib
-- Library libssl found C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libssl.lib
-- Skipping already existing target: CONAN_LIB::OpenSSL_SSL_libssl
-- Found: C:/Users/Dhruv/.conan/data/openssl/1.1.1h/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libssl.lib
-- Could NOT find WrapOpenSSLHeaders (missing: WrapOpenSSLHeaders_FOUND)
-- Found WrapOpenSSL: TRUE
-- Performing Test HAVE_openssl
-- Performing Test HAVE_openssl - Success
-- Could NOT find GSSAPI (missing: GSSAPI_LIBRARIES GSSAPI_INCLUDE_DIRS)
-- Performing Test HAVE_getifaddrs
-- Performing Test HAVE_getifaddrs - Failed
-- Performing Test HAVE_ipv6ifname
-- Performing Test HAVE_ipv6ifname - Failed
-- Performing Test HAVE_linux_netlink
-- Performing Test HAVE_linux_netlink - Failed
-- Performing Test HAVE_sctp
-- Performing Test HAVE_sctp - Failed
-- Performing Test HAVE_netlistmgr
-- Performing Test HAVE_netlistmgr - Success
-- FindLibdrm.cmake cannot find libdrm on Windows systems.
-- Performing Test HAVE_EGL
-- Performing Test HAVE_EGL - Failed
-- Could NOT find EGL (missing: EGL_INCLUDE_DIR HAVE_EGL EGL_LIBRARY)
-- Conan: Using autogenerated FindFreetype.cmake
-- Library freetype found C:/Users/Dhruv/.conan/data/freetype/2.10.2/_/_/package/2eda287fd36b7b010dea069857045000246077e3/lib/freetype.lib
-- Found: C:/Users/Dhruv/.conan/data/freetype/2.10.2/_/_/package/2eda287fd36b7b010dea069857045000246077e3/lib/freetype.lib
-- Conan: Using autogenerated FindPNG.cmake
-- Library libpng16 found C:/Users/Dhruv/.conan/data/libpng/1.6.37/_/_/package/d140711d95cc16a85766a8fc3a551dfafe84cf63/lib/libpng16.lib
-- Found: C:/Users/Dhruv/.conan/data/libpng/1.6.37/_/_/package/d140711d95cc16a85766a8fc3a551dfafe84cf63/lib/libpng16.lib
-- Dependency ZLIB already found
-- Dependency ZLIB already found
-- Conan: Using autogenerated FindBZip2.cmake
-- Library bz2 found C:/Users/Dhruv/.conan/data/bzip2/1.0.8/_/_/package/d16a91eadaaf5829b928b12d2f836ff7680d3df5/lib/bz2.lib
-- Found: C:/Users/Dhruv/.conan/data/bzip2/1.0.8/_/_/package/d16a91eadaaf5829b928b12d2f836ff7680d3df5/lib/bz2.lib
-- Found WrapSystemFreetype: TRUE (found suitable version "2.10.2", minimum required is "2.2.0")
-- Could NOT find Fontconfig (missing: Fontconfig_LIBRARY Fontconfig_INCLUDE_DIR)
-- Findgbm.cmake cannot find gbm on Windows systems.
-- Conan: Using autogenerated Findharfbuzz.cmake
-- Library harfbuzz found C:/.conan/2ecaf5/1/lib/harfbuzz.lib
-- Found: C:/.conan/2ecaf5/1/lib/harfbuzz.lib
-- Dependency Freetype already found
-- Conan: Using autogenerated Findglib.cmake
-- Library gthread-2.0 found C:/.conan/ba70c5/1/lib/gthread-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/gthread-2.0.lib
-- Library gio-2.0 found C:/.conan/ba70c5/1/lib/gio-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/gio-2.0.lib
-- Library gmodule-2.0 found C:/.conan/ba70c5/1/lib/gmodule-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/gmodule-2.0.lib
-- Library gobject-2.0 found C:/.conan/ba70c5/1/lib/gobject-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/gobject-2.0.lib
-- Library glib-2.0 found C:/.conan/ba70c5/1/lib/glib-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/glib-2.0.lib
-- Conan: Dependency ZLIB already found
-- Conan: Using autogenerated Findlibffi.cmake
-- Library libffi found C:/Users/Dhruv/.conan/data/libffi/3.3/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libffi.lib
-- Found: C:/Users/Dhruv/.conan/data/libffi/3.3/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libffi.lib
-- Conan: Using autogenerated FindPCRE.cmake
-- Library pcreposix found C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcreposix.lib
-- Found: C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcreposix.lib
-- Library pcre found C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre.lib
-- Found: C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre.lib
-- Library pcre16 found C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre16.lib
-- Found: C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre16.lib
-- Library pcre32 found C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre32.lib
-- Found: C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre32.lib
-- Conan: Dependency BZip2 already found
-- Conan: Dependency ZLIB already found
-- Library pcre32 found C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre32.lib
-- Found: C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre32.lib
-- Library pcre16 found C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre16.lib
-- Found: C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre16.lib
-- Library pcre found C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre.lib
-- Found: C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcre.lib
-- Library pcreposix found C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcreposix.lib
-- Found: C:/Users/Dhruv/.conan/data/pcre/8.44/_/_/package/dc9afb8c60d4a368d09c45c18fcf7d7b996c7866/lib/pcreposix.lib
-- Conan: Using autogenerated Findlibgettext.cmake
-- Library gnuintl found C:/Users/Dhruv/.conan/data/libgettext/0.20.1/_/_/package/b13ada78896add21999e422ef79cd1de53c4d4f5/lib/gnuintl.lib
-- Found: C:/Users/Dhruv/.conan/data/libgettext/0.20.1/_/_/package/b13ada78896add21999e422ef79cd1de53c4d4f5/lib/gnuintl.lib
-- Conan: Using autogenerated FindIconv.cmake
-- Library iconv found C:/Users/Dhruv/.conan/data/libiconv/1.16/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/iconv.lib
-- Found: C:/Users/Dhruv/.conan/data/libiconv/1.16/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/iconv.lib
-- Library charset found C:/Users/Dhruv/.conan/data/libiconv/1.16/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/charset.lib
-- Found: C:/Users/Dhruv/.conan/data/libiconv/1.16/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/charset.lib
-- Conan: Using autogenerated Findlibelf.cmake
-- Library elf found C:/Users/Dhruv/.conan/data/libelf/0.8.13/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/elf.lib
-- Found: C:/Users/Dhruv/.conan/data/libelf/0.8.13/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/elf.lib
-- Library glib-2.0 found C:/.conan/ba70c5/1/lib/glib-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/glib-2.0.lib
-- Library gobject-2.0 found C:/.conan/ba70c5/1/lib/gobject-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/gobject-2.0.lib
-- Library gmodule-2.0 found C:/.conan/ba70c5/1/lib/gmodule-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/gmodule-2.0.lib
-- Library gio-2.0 found C:/.conan/ba70c5/1/lib/gio-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/gio-2.0.lib
-- Library gthread-2.0 found C:/.conan/ba70c5/1/lib/gthread-2.0.lib
-- Found: C:/.conan/ba70c5/1/lib/gthread-2.0.lib
-- Found WrapSystemHarfbuzz: TRUE (found suitable version "2.6.8", minimum required is "2.6.0")
-- FindLibinput.cmake cannot find libinput on Windows systems.
-- Conan: Using autogenerated FindJPEG.cmake
-- Found JPEG: 9d (found version "9d")
-- Library libjpeg found C:/Users/Dhruv/.conan/data/libjpeg/9d/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libjpeg.lib
-- Found: C:/Users/Dhruv/.conan/data/libjpeg/9d/_/_/package/3fb49604f9c2f729b85ba3115852006824e72cab/lib/libjpeg.lib
-- Conan: Using autogenerated FindPNG.cmake
-- Library libpng16 found C:/Users/Dhruv/.conan/data/libpng/1.6.37/_/_/package/d140711d95cc16a85766a8fc3a551dfafe84cf63/lib/libpng16.lib
-- Skipping already existing target: CONAN_LIB::PNG_libpng16
-- Found: C:/Users/Dhruv/.conan/data/libpng/1.6.37/_/_/package/d140711d95cc16a85766a8fc3a551dfafe84cf63/lib/libpng16.lib
-- Found WrapSystemPNG: C:/Users/Dhruv/.conan/data/libpng/1.6.37/_/_/package/d140711d95cc16a85766a8fc3a551dfafe84cf63/lib/libpng16.lib;ZLIB::ZLIB (found version "1.6.37")
-- Found OpenGL: opengl32
-- Found WrapOpenGL: TRUE
-- Could NOT find EGL (missing: EGL_INCLUDE_DIR HAVE_EGL EGL_LIBRARY)
-- Performing Test HAVE_GLESv2
-- Performing Test HAVE_GLESv2 - Failed
-- Could NOT find GLESv2 (missing: GLESv2_INCLUDE_DIR GLESv2_LIBRARY HAVE_GLESv2 HAVE_GLESv2)
-- Could NOT find Vulkan (missing: Vulkan_LIBRARY Vulkan_INCLUDE_DIR)
-- Performing Test HAVE_evdev
-- Performing Test HAVE_evdev - Failed
-- Performing Test HAVE_integrityfb
-- Performing Test HAVE_integrityfb - Failed
-- Performing Test HAVE_linuxfb
-- Performing Test HAVE_linuxfb - Failed
-- Performing Test HAVE_directwrite
-- Performing Test HAVE_directwrite - Success
-- Performing Test HAVE_directwrite3
-- Performing Test HAVE_directwrite3 - Success
-- Performing Test HAVE_d2d1
-- Performing Test HAVE_d2d1 - Success
-- Performing Test HAVE_d2d1_1
-- Performing Test HAVE_d2d1_1 - Success
-- Tool 'Qt6::moc' will be built from source.
-- Tool 'Qt6::rcc' will be built from source.
-- Tool 'Qt6::tracegen' will be built from source.
-- Tool 'Qt6::cmake_automoc_parser' will be built from source.
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE
-- Using system PCRE2.
-- Found WrapPCRE2: TRUE
CMake Error at qtbase/cmake/QtSyncQtHelpers.cmake:7 (message):
  Perl needs to be available to build Qt.
Call Stack (most recent call first):
  qtbase/cmake/QtSyncQtHelpers.cmake:12 (qt_ensure_perl)
  qtbase/cmake/QtModuleHelpers.cmake:168 (qt_ensure_sync_qt)
  qtbase/src/corelib/CMakeLists.txt:29 (qt_internal_add_module)


-- Configuring incomplete, errors occurred!
```
as can be observed the configuration step fails to find the `ZSTD` and `double-conversion` packages

based on the patch in https://bugreports.qt.io/browse/QTBUG-88091?jql=labels%20%3D%20cmake_qt6
our configuration still doesn't work for either `zstd` nor `double-conversion` since from what I observed they expect `find_packageConfiguration` and we don't have/use the generator for that.

**Comment by Dark-Passenger** [comment]
Addendum: as can be observed regular find_package calls do indeed succeed e.g. `ZLIB`.

**Comment by ericLemanissier** [comment]
Thanks ! What is this `find_packageConfiguration` you are talking about ? it does not appear in the patch you linked right ?
Regarding the last failure in your log (the only blocking one), I reported it on https://bugreports.qt.io/browse/QTBUG-87480 and you can probably workaround it by adding a build_requirement on `strawberryperl/5.30.0.1`

**Comment by Dark-Passenger** [comment]
> Thanks ! What is this `find_packageConfiguration` you are talking about ? it does not appear in the patch you linked right ?

`find_package(ZSTD CONFIG)` the `find_package` call looks for `FindZSTDConfig.cmake` which doesn't exist in the build directory.

> Regarding the last failure in your log (the only blocking one), I reported it on https://bugreports.qt.io/browse/QTBUG-87480 and you can probably workaround it by adding a build_requirement on `strawberryperl/5.30.0.1`

yup, I realized that too 😁

**Comment by ericLemanissier** [comment]
ok, so it's probably just a matter of adding `cmake_find_package_multi` in the generators attribute

**Comment by Dark-Passenger** [comment]
> ok, so it's probably just a matter of adding `cmake_find_package_multi` in the generators attribute

oh does that generator create the config file ? that should really be documented (but this is a packager problem only.

**Comment by ericLemanissier** [comment]
yes : https://docs.conan.io/en/latest/reference/generators/cmake_find_package_multi.html#generated-files

**Comment by Dark-Passenger** [comment]
> yes : https://docs.conan.io/en/latest/reference/generators/cmake_find_package_multi.html#generated-files

serves me right for not reading 😅

**Comment by Dark-Passenger** [comment]
> Regarding the last failure in your log (the only blocking one), I reported it on https://bugreports.qt.io/browse/QTBUG-87480 and you can probably workaround it by adding a build_requirement on strawberryperl/5.30.0.1

strawberryperl is a horrible dependancy, it installs gcc and basically breaks compiler detection on windows.

**Comment by ericLemanissier** [comment]
I don't know of any alternive for perl than to install manually https://www.activestate.com/products/perl/downloads/
The dependency detection issue should be fixed in `qt/6.0.0@bincrafters/stable` which is being compiled and uploaded currently.
Please report if you still have some issues
